llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -1578,6 +1578,11 @@ class MovDPP8Pattern<Predicate Pred, Instruction Inst, ValueType vt> : GCNPat <
   let OtherPredicates = [Pred];
 }
 
+def : GCNPat <
```

Comment:
Can you do this in the definition of V_CVT_OFF_F32_I4, instead of using a separate Pat?

Reply:
I'm not fully familiar with TableGen syntax.

I'm having trouble to put this in the definition since the pattern uses the definition of `V_CVT_OFF_F32_I4`.

Reply:
Thanks! This worked